[2023-08-30T18:56:37.302+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.314+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T18:56:37.326+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T18:56:37.328+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T18:56:37.342+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T18:56:37.349+0000] {standard_task_runner.py:57} INFO - Started process 501 to run task
[2023-08-30T18:56:37.356+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpa75t7dfs']
[2023-08-30T18:56:37.360+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask install_required_packages
[2023-08-30T18:56:37.425+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.554+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host 5e7fbf8b7901
[2023-08-30T18:56:37.594+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:37.713+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T18:56:37.716+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T18:56:37.718+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T18:56:37.743+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T18:56:39.315+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T18:56:40.031+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T18:56:40.111+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T18:56:40.121+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T18:56:40.139+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T18:56:40.143+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T18:56:40.146+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T18:56:40.152+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T18:56:43.766+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T18:56:43.850+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T18:56:44.256+0000] {subprocess.py:93} INFO - 
[2023-08-30T18:56:44.257+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T18:56:44.258+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T18:56:44.528+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T18:56:44.586+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T185637, end_date=20230830T185644
[2023-08-30T18:56:44.630+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T18:56:44.654+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:19:23.803+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:23.812+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:19:23.822+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:19:23.824+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:19:23.840+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T19:19:23.847+0000] {standard_task_runner.py:57} INFO - Started process 216 to run task
[2023-08-30T19:19:23.859+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpts979pnz']
[2023-08-30T19:19:23.862+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask install_required_packages
[2023-08-30T19:19:23.920+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:24.025+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host 884d86529f65
[2023-08-30T19:19:24.053+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:19:24.182+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T19:19:24.187+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:19:24.191+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:19:24.208+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:19:28.625+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:19:29.494+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:19:29.590+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:19:29.606+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:19:29.614+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:19:29.616+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:19:29.618+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:19:29.620+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:19:57.266+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:19:57.396+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:19:57.713+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:19:57.715+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:19:57.717+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:19:57.991+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:19:58.082+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T191923, end_date=20230830T191958
[2023-08-30T19:19:58.151+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:19:58.200+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:31:06.367+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.377+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:31:06.386+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:31:06.387+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:31:06.407+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T19:31:06.415+0000] {standard_task_runner.py:57} INFO - Started process 218 to run task
[2023-08-30T19:31:06.422+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpo_daqlto']
[2023-08-30T19:31:06.425+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask install_required_packages
[2023-08-30T19:31:06.492+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.609+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host f6bcf103ff79
[2023-08-30T19:31:06.638+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:06.784+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T19:31:06.797+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:31:06.800+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:31:06.825+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:31:08.991+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:31:11.142+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:31:11.271+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:31:11.284+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:31:11.292+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:31:11.298+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:31:11.299+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:31:11.305+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:31:32.938+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:31:33.110+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:31:33.240+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:31:33.242+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:31:33.242+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:31:33.516+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:31:33.639+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T193106, end_date=20230830T193133
[2023-08-30T19:31:33.678+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:31:33.727+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:41:18.492+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.503+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:41:18.516+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:41:18.519+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:41:18.538+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T19:41:18.551+0000] {standard_task_runner.py:57} INFO - Started process 278 to run task
[2023-08-30T19:41:18.557+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp02phj3ag']
[2023-08-30T19:41:18.561+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask install_required_packages
[2023-08-30T19:41:18.606+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.719+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host 81eeb4b7c8a9
[2023-08-30T19:41:18.753+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:18.878+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T19:41:18.895+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:41:18.897+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T19:41:18.919+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:41:32.762+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:41:33.418+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:41:33.835+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:41:33.876+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:41:33.894+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:41:33.897+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:41:33.909+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:41:33.911+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:41:49.337+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy
[2023-08-30T19:41:49.393+0000] {subprocess.py:93} INFO - Successfully installed sodapy-2.2.0
[2023-08-30T19:41:49.405+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:41:49.406+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:41:49.407+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:41:49.693+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:41:49.751+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T194118, end_date=20230830T194149
[2023-08-30T19:41:49.786+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:41:49.825+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:50:05.872+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:05.884+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:50:05.903+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:50:05.905+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:50:05.934+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T19:50:05.952+0000] {standard_task_runner.py:57} INFO - Started process 242 to run task
[2023-08-30T19:50:05.969+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpywxc9ugl']
[2023-08-30T19:50:05.992+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask install_required_packages
[2023-08-30T19:50:10.930+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:11.036+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host 206ecb1dc8df
[2023-08-30T19:50:11.082+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:11.595+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T19:50:11.729+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:50:11.759+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T19:50:11.840+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:50:25.575+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:50:30.717+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:50:30.872+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:50:31.239+0000] {subprocess.py:93} INFO - Collecting geoalchemy2
[2023-08-30T19:50:31.259+0000] {subprocess.py:93} INFO -   Downloading GeoAlchemy2-0.14.1-py3-none-any.whl (54 kB)
[2023-08-30T19:50:31.297+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.0/55.0 kB 1.5 MB/s eta 0:00:00
[2023-08-30T19:50:31.306+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:50:31.312+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T19:50:31.313+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T19:50:31.326+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:50:31.330+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:50:31.332+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:50:31.334+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:50:31.423+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T19:50:50.050+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy, geoalchemy2
[2023-08-30T19:50:50.168+0000] {subprocess.py:93} INFO - Successfully installed geoalchemy2-0.14.1 sodapy-2.2.0
[2023-08-30T19:50:50.180+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:50:50.181+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:50:50.182+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:50:50.364+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:50:50.419+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T195005, end_date=20230830T195050
[2023-08-30T19:50:50.472+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:50:50.505+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:54:01.066+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.079+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:54:01.089+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T19:54:01.091+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:54:01.108+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T19:54:01.115+0000] {standard_task_runner.py:57} INFO - Started process 202 to run task
[2023-08-30T19:54:01.121+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpg0fwq7d7']
[2023-08-30T19:54:01.128+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask install_required_packages
[2023-08-30T19:54:01.204+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.317+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host 86f9e97d6c70
[2023-08-30T19:54:01.366+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:01.485+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T19:54:01.497+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:54:01.500+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T19:54:01.523+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:54:04.604+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T19:54:05.181+0000] {subprocess.py:93} INFO - Collecting sodapy
[2023-08-30T19:54:05.272+0000] {subprocess.py:93} INFO -   Downloading sodapy-2.2.0-py2.py3-none-any.whl (15 kB)
[2023-08-30T19:54:05.321+0000] {subprocess.py:93} INFO - Collecting geoalchemy2
[2023-08-30T19:54:05.622+0000] {subprocess.py:93} INFO -   Downloading GeoAlchemy2-0.14.1-py3-none-any.whl (54 kB)
[2023-08-30T19:54:05.670+0000] {subprocess.py:93} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 55.0/55.0 kB 1.3 MB/s eta 0:00:00
[2023-08-30T19:54:05.681+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T19:54:05.687+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T19:54:05.694+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T19:54:05.700+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T19:54:05.703+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T19:54:05.708+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T19:54:05.713+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T19:54:05.797+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T19:54:23.638+0000] {subprocess.py:93} INFO - Installing collected packages: sodapy, geoalchemy2
[2023-08-30T19:54:23.736+0000] {subprocess.py:93} INFO - Successfully installed geoalchemy2-0.14.1 sodapy-2.2.0
[2023-08-30T19:54:23.748+0000] {subprocess.py:93} INFO - 
[2023-08-30T19:54:23.749+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T19:54:23.750+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T19:54:24.013+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:54:24.076+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T195401, end_date=20230830T195424
[2023-08-30T19:54:24.190+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:54:24.235+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:05:28.670+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:28.680+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:05:28.689+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:05:28.691+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:05:28.713+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T20:05:28.717+0000] {standard_task_runner.py:57} INFO - Started process 419 to run task
[2023-08-30T20:05:28.721+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpz235q4r6']
[2023-08-30T20:05:28.725+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask install_required_packages
[2023-08-30T20:05:28.752+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:28.821+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host 3938d7abd9ff
[2023-08-30T20:05:28.836+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:28.986+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T20:05:29.006+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:05:29.007+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T20:05:29.049+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:05:41.595+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:05:41.855+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:05:41.856+0000] {subprocess.py:93} INFO - Requirement already satisfied: geoalchemy2 in /home/***/.local/lib/python3.8/site-packages (0.14.1)
[2023-08-30T20:05:41.859+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:05:41.863+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T20:05:41.865+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T20:05:41.870+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:05:41.871+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:05:41.872+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:05:41.874+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:05:41.925+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T20:05:45.558+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:05:45.562+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:05:45.565+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:05:45.897+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:05:45.936+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T200528, end_date=20230830T200545
[2023-08-30T20:05:45.980+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:05:46.003+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:33:19.760+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:19.775+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:33:19.788+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:33:19.790+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:33:19.809+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T20:33:19.816+0000] {standard_task_runner.py:57} INFO - Started process 233 to run task
[2023-08-30T20:33:19.847+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpqguxe7p4']
[2023-08-30T20:33:19.858+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask install_required_packages
[2023-08-30T20:33:19.923+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:20.009+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host 961f38e8363e
[2023-08-30T20:33:20.032+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:20.245+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T20:33:20.284+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:33:20.307+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy geoalchemy2']
[2023-08-30T20:33:20.335+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:33:33.784+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:33:39.278+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:33:39.279+0000] {subprocess.py:93} INFO - Requirement already satisfied: geoalchemy2 in /home/***/.local/lib/python3.8/site-packages (0.14.1)
[2023-08-30T20:33:39.282+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:33:39.291+0000] {subprocess.py:93} INFO - Requirement already satisfied: SQLAlchemy>=1.4 in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (1.4.49)
[2023-08-30T20:33:39.295+0000] {subprocess.py:93} INFO - Requirement already satisfied: packaging in /home/***/.local/lib/python3.8/site-packages (from geoalchemy2) (23.1)
[2023-08-30T20:33:39.301+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:33:39.303+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:33:39.304+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:33:39.311+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:33:39.389+0000] {subprocess.py:93} INFO - Requirement already satisfied: greenlet!=0.4.17 in /home/***/.local/lib/python3.8/site-packages (from SQLAlchemy>=1.4->geoalchemy2) (2.0.2)
[2023-08-30T20:33:45.651+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:33:45.669+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:33:45.715+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:33:46.537+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:33:46.622+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T203319, end_date=20230830T203346
[2023-08-30T20:33:46.669+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:33:46.699+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:48:15.022+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:15.032+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:48:15.056+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T20:48:15.057+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:48:15.080+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T20:48:15.086+0000] {standard_task_runner.py:57} INFO - Started process 295 to run task
[2023-08-30T20:48:15.096+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpzt5wbnhz']
[2023-08-30T20:48:15.106+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask install_required_packages
[2023-08-30T20:48:15.152+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:15.229+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host 8f66711a2898
[2023-08-30T20:48:15.254+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:15.330+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T20:48:15.332+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:48:15.335+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T20:48:15.359+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:48:16.412+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T20:48:16.873+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T20:48:16.878+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T20:48:16.888+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T20:48:16.891+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T20:48:16.895+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T20:48:16.897+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T20:48:21.096+0000] {subprocess.py:93} INFO - 
[2023-08-30T20:48:21.098+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T20:48:21.100+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T20:48:21.315+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:48:21.356+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T204815, end_date=20230830T204821
[2023-08-30T20:48:21.384+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:48:21.414+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:07:24.058+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:24.068+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T21:07:24.083+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T21:07:24.085+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:07:24.100+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T21:07:24.117+0000] {standard_task_runner.py:57} INFO - Started process 261 to run task
[2023-08-30T21:07:24.142+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp8vk4y4s0']
[2023-08-30T21:07:24.154+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask install_required_packages
[2023-08-30T21:07:24.208+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:24.330+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host cbc745b7550b
[2023-08-30T21:07:24.351+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:24.460+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T21:07:24.476+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:07:24.477+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T21:07:24.504+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:07:26.246+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T21:07:27.054+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T21:07:27.058+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T21:07:27.063+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T21:07:27.065+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T21:07:27.067+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T21:07:27.068+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T21:07:33.031+0000] {subprocess.py:93} INFO - 
[2023-08-30T21:07:33.033+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T21:07:33.038+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T21:07:33.306+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:07:33.394+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T210724, end_date=20230830T210733
[2023-08-30T21:07:33.440+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:07:33.475+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:16:37.086+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:37.097+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T21:16:37.123+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [queued]>
[2023-08-30T21:16:37.124+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:16:37.144+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): install_required_packages> on 2023-08-27 00:00:00+00:00
[2023-08-30T21:16:37.150+0000] {standard_task_runner.py:57} INFO - Started process 303 to run task
[2023-08-30T21:16:37.159+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'install_required_packages', 'scheduled__2023-08-27T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpxag6td8r']
[2023-08-30T21:16:37.172+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask install_required_packages
[2023-08-30T21:16:37.203+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:37.264+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.install_required_packages scheduled__2023-08-27T00:00:00+00:00 [running]> on host 5b9f14bf20e0
[2023-08-30T21:16:37.295+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:16:37.399+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='install_required_packages' AIRFLOW_CTX_EXECUTION_DATE='2023-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-27T00:00:00+00:00'
[2023-08-30T21:16:37.404+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:16:37.406+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pip install sodapy']
[2023-08-30T21:16:37.438+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:16:48.047+0000] {subprocess.py:93} INFO - Defaulting to user installation because normal site-packages is not writeable
[2023-08-30T21:16:49.904+0000] {subprocess.py:93} INFO - Requirement already satisfied: sodapy in /home/***/.local/lib/python3.8/site-packages (2.2.0)
[2023-08-30T21:16:49.919+0000] {subprocess.py:93} INFO - Requirement already satisfied: requests>=2.28.1 in /home/***/.local/lib/python3.8/site-packages (from sodapy) (2.31.0)
[2023-08-30T21:16:49.928+0000] {subprocess.py:93} INFO - Requirement already satisfied: certifi>=2017.4.17 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (2023.7.22)
[2023-08-30T21:16:49.930+0000] {subprocess.py:93} INFO - Requirement already satisfied: charset-normalizer<4,>=2 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.2.0)
[2023-08-30T21:16:49.932+0000] {subprocess.py:93} INFO - Requirement already satisfied: urllib3<3,>=1.21.1 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (1.26.16)
[2023-08-30T21:16:49.934+0000] {subprocess.py:93} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.8/site-packages (from requests>=2.28.1->sodapy) (3.4)
[2023-08-30T21:16:59.919+0000] {subprocess.py:93} INFO - 
[2023-08-30T21:16:59.922+0000] {subprocess.py:93} INFO - [notice] A new release of pip is available: 23.0.1 -> 23.2.1
[2023-08-30T21:16:59.924+0000] {subprocess.py:93} INFO - [notice] To update, run: pip install --upgrade pip
[2023-08-30T21:17:00.365+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:17:00.418+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=install_required_packages, execution_date=20230827T000000, start_date=20230830T211637, end_date=20230830T211700
[2023-08-30T21:17:00.486+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:17:00.536+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
